<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="class_listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>150</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="notice_listWidget">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>200</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>200</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>새로운 메시지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="class_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>150</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>나의 수강 목록</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>670</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="class_label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>670</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>사이버 캠퍼스 진행 현황</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
